spec/controllers/tasks_controller_spec.rb

```diff
@@ -14,4 +14,11 @@
             expect(JSON.parse(response.body)).to eq("error" => "The task couldn't be added. Try again!")
         end
     end 
+
+    describe "GET#index" do
+        it" returns http success " do
```

Comment:
What's with this weird spacing? 🧐

Reply:
Do you mean line 17? 🤔 I let them to distinguish between contexts.In my VSCode looks nice😊 but I can get rid of them.

Reply:
No line 19. Why is it like `it" returns http success "` and not `it "returns http success"`?

Reply:
Upsss...space invaders 👾 ...Fix them!